### PR TITLE
Add Finnish/Swedish keyboard layout

### PR DESCRIPTION
**Pull request by SPuntte**
See #36.

For reference:

Fujitsu KB410
Fujitsu KB410

Dell RT7D60
Dell RT7D60

### CODE REVIEW

File: src/layouts/mod.rs [hunk 1]
```diff
@@ -31,6 +31,9 @@ pub use self::de105::De105Key;
 mod no105;
 pub use self::no105::No105Key;
 
+mod fi_se105;
+pub use self::fi_se105::FiSe105Key;
+
 /// A enum of all the supported keyboard layouts.
 pub enum AnyLayout {
```
Comment (jonathanpallant) [inline src/layouts/mod.rs:38]: Please add here.

Comment (SPuntte) [reply src/layouts/mod.rs:38]: Oof, missed that one. Thanks! :)

**Comment by jonathanpallant** [comment]
Looks OK to me, but please also add to the README file and to the CHANGELOG file. Thank you!

**Review by eldruin — APPROVED** [review]
Looks good to me, thank you!

**Pull request merged**